app/domain/audits/policies/non_allocated.rb

```diff
@@ -0,0 +1,9 @@
+module Audits
+  module Policies
+    class NonAllocated
```

Comment:
I think `Unallocated` would be the correct name to use here based on [the dictionary](http://www.dictionary.com/browse/unallocated)!